addons/common/functions/fnc_claim.sqf

```diff
@@ -30,16 +30,18 @@ _target setVariable [QGVAR(owner), _unit, true];
 
 // lock target object
 if (_lockTarget) then {
-    private _canBeDisassembled = ([] isNotEqualTo getArray (configOf _target >> "assembleInfo" >> "dissasembleTo")) && { !([false, true] select (_target getVariable [QEGVAR(csw,assemblyMode), 0])) };
```

Comment:
assemblyMode can go from 0-3 so I think this is/was broken?

Reply:
Probably?